modules/distribution/carbon-home/conf/editor/deployment.yaml

```diff
@@ -0,0 +1,205 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```

Comment:
Year should be 2017 in the license header

Reply:
fixed